<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>880</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="MyGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>70</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButtonAddSourceImage">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Add source image</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Camera.png</normaloff>:/Icons/Camera.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAddSprite">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Add sprite</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Plus.png</normaloff>:/Icons/Plus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonDeleteSprite">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Delete sprite/image tree</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Minus.png</normaloff>:/Icons/Minus.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCog">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Enable centre of gravity</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/Icons/Cog.png</normaloff>:/Icons/Cog.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>880</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionBuild"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionBuild"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/NewFile.png</normaloff>:/Icons/NewFile.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/FileSave.png</normaloff>:/Icons/FileSave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/FileOpen.png</normaloff>:/Icons/FileOpen.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save As</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/FileClose.png</normaloff>:/Icons/FileClose.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Icons/Build.png</normaloff>:/Icons/Build.png</iconset>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header location="global">mygraphicsview.h</header>
   <slots>
    <signal>dataChanged()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButtonAddSourceImage</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addSourceImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>628</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemClicked(QTreeWidgetItem*,int)</signal>
   <receiver>MainWindow</receiver>
   <slot>itemClicked(QTreeWidgetItem*,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonAddSprite</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>addSprite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>662</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDeleteSprite</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>deleteSprite()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>696</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeWidget</sender>
   <signal>itemSelectionChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>itemSelectionChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>741</x>
     <y>303</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCog</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>pushButtonCogClicked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>730</x>
     <y>58</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNew</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileNew()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionOpen</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileOpen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_As</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileSaveAs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>fileClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphicsView</sender>
   <signal>dataChanged()</signal>
   <receiver>MainWindow</receiver>
   <slot>dataChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBuild</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>build()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>439</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSourceImage()</slot>
  <slot>itemClicked(QTreeWidgetItem*,int)</slot>
  <slot>addSprite()</slot>
  <slot>deleteSprite()</slot>
  <slot>itemSelectionChanged()</slot>
  <slot>pushButtonCogClicked(bool)</slot>
  <slot>fileOpen()</slot>
  <slot>fileNew()</slot>
  <slot>fileSave()</slot>
  <slot>fileSaveAs()</slot>
  <slot>fileClose()</slot>
  <slot>dataChanged()</slot>
  <slot>build()</slot>
 </slots>
</ui>
